src/components/atoms/Button/Component.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledButton = styled.button<Props>`
+  display: flex;
+  border-radius: 4px;
+  border: 0;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+
+  ${(props) =>
+    props.buttonType === 'primary' &&
+    `background: rgba(47, 128, 237, 1);
+      color: white;
+      width: 260px;
+      height: 38px;
+      font-weight: 700;
+      font-size: 14px;
+      
+      &:active {
+        background: rgba(47, 128, 237, .15)
+      }`}
+
+  ${(props) =>
+    props.buttonType === 'secondary' &&
+    `background: rgba(47, 128, 237, .15);
+      color: #2F80ED;
+      width: 56px;
+      height: 32px;
+      font-size: 14px;
+      font-weight: 400;`}
+
+  ${(props) =>
+    props.buttonType === 'warning' &&
+    `background: rgba(252, 224, 224, 1);
+      color: #ED2F2F;
+      width: 80px;
+      height: 38px;
+      font-size: 14px;
+      font-weight: 400;`}
+`;
+
+export type Props = {
+  buttonType?: 'primary' | 'secondary' | 'warning';
+  label?: string;
+  onClick?: () => void;
+};
+
+const Component = ({
+  buttonType,
+  label,
+  onClick,
+  ...props
```

Comment:
Please add a trailing comma here @jhonangelo 

Trailing commas are a code-style convention that was introduced to avoid spurious differences in version controls (ie Git).

Imagine you have version controlled code and you have to change it. When you add a new line to your object without the trailing comma, you will have to change the original last line by adding a comma. In version control this then shows up as two changed lines. The code reviewer or a future dev have to check if you effectively changed the last line, or only added the comma.

Reply:
No need to add here since spreadable does not need a trailing comma